### ISSUE TITLE
Text to EV3 display

### ISSUE THREAD

**Issue by daanjels** [arqetype/Lejos#15]
Create a helper class to easily flow text on the EV3 display. Several scenarios are possible:
1 a method that flows a string into the display, this would be arbitrary, but very easy for simple messaging.
2 a separate 'application' that displays a window with a box for text input. A reflow button to adjust the text to the screen. And a 'code' button that creates the code to use in a leJOS program. Nice to copy-edit text for the EV3 screen.
3 a display tool for longer texts that allow scrolling.

**Comment by daanjels** [comment]
Built a class that chops up a string into pieces that are maximum 18 characters. It splits on spaces and dashes. Javacode for EV3 is rendered in the console, to copy and paste.
The next step is to build this method into a LeJOS project so the code will be executed, displaying the text correctly on the EV3 screen.
This means I will have to check the length of the supplied string, ultimately displaying scrollbuttons if it is longer than the screen can display?

**Comment by daanjels** [comment]
reviewed and committed the basic code (not the leJOS version)

**Comment by daanjels** [comment]
Created a package 'Display' to create an EV3 program that flows a string on the LCD wrapping words on the next line. It uses the method 'wordWrap', which chops up any given string into pieces that fit the width of the screen. End-of-line markers (\n) are respected for the final output. If a string is too long to fit on the 8 lines of the screen, an arrow is shown and the down button will scroll through the text.
Next step: figure out if I can extend the LCD class with this method so it can be used in any EV3 program

**Comment by daanjels** [comment]
Extending the LCD class is not possible (the LCD class uses a private constructor).
So I created a separate (empty) class to store the method. I've put the 'Screen' class in a separate package 'myEV3' so this method can be used in all of my programs.
Updated the code to support words longer then 18 characters without dash: the will break at the end of the line.
Also adjusted the arrow display. When scrolling, the arrow is removed.
